lib/_http_client.js

```diff
@@ -283,7 +274,7 @@ ClientRequest.prototype._implicitHeader = function _implicitHeader() {
 
 ClientRequest.prototype.abort = function abort() {
   if (!this.aborted) {
-    process.nextTick(emitAbortNT, this);
+    process.nextTick(emitAbortNT.bind(this));
```

Comment:
Yep, I know. I'm fine with these not being backported or only being partially backported.